src/app/public/modules/inline-delete/inline-delete.component.ts

```diff
@@ -89,15 +96,15 @@ export class SkyInlineDeleteComponent implements OnDestroy, OnInit {
   @Output()
   public deleteTriggered = new EventEmitter<void>();
 
-  @ViewChild('delete')
-  public deleteButton: ElementRef;
-
   public animationState: string = 'shown';
 
   public assistiveTextId: string = `sky-inline-delete-assistive-text-${++nextId}`;
 
   public type: SkyInlineDeleteType = SkyInlineDeleteType.Standard;
 
+  @ViewChild('delete')
+  private deleteButton: ElementRef;
```

Comment:
No reason for this view child to be public.

Reply:
Right on. Child privacy is serious business.